win32ss/gdi/ntgdi/freetype.c

```diff
@@ -1778,7 +1778,7 @@ IntGdiAddFontResourceEx(PUNICODE_STRING FileName, DWORD Characteristics,
 INT FASTCALL
 IntGdiAddFontResource(PUNICODE_STRING FileName, DWORD Characteristics)
 {
-    return IntGdiAddFontResourceEx(FileName, Characteristics, AFRX_DOS_DEVICE_PATH);
+    return IntGdiAddFontResourceEx(FileName, Characteristics, 0);
```

Comment:
Do you still use that (removed) flag elsewhere in the code?

Reply:
Yes.